infra/helper.py

```diff
@@ -342,7 +342,10 @@ def _workdir_from_dockerfile(project_name):
 
 def docker_run(run_args, print_output=True):
   """Call `docker run`."""
-  command = ['docker', 'run', '--rm', '-i', '--privileged']
+  command = ['docker', 'run', '--rm', '--privileged']
+  # Support environments with a TTY.
+  if sys.stdin.isatty():
+    command.append('-i')
```

Comment:
blank line after this line and before 346

Reply:
The reason why I asked for this here and above is to improve readability by separating logical sections of code visually.

Reply:
Done.